### PR TITLE
Instructions and questions page

### PR DESCRIPTION
**Pull request by darshandamre**
Created instructions and questions page.
[Preview instructions page](https://quiz-mini-project-git-instructions-and-ques-076847-darshandamre.vercel.app/pages/instructions.html)
[Preview questions page](https://quiz-mini-project-git-instructions-and-ques-076847-darshandamre.vercel.app/pages/question.html)
[Preview answers](https://quiz-mini-project-git-instructions-and-ques-076847-darshandamre.vercel.app/pages/answer.html)

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
This file looks good to me , naming convention is proper

File: style.css [hunk 1]
```diff
@@ -52,3 +58,26 @@ a {
 .card {
   max-width: none;
 }
+
+.question-data {
+  display: flex;
+  justify-content: space-between;
```
Comment (arjunbharti) [inline style.css:64]: If you have created utilities classes in your component library then you can use them there otherwise it's fine!

**Review by arjunbharti — APPROVED** [review]
Added a minor suggestion. Everything else is fine!

File: style.css (hunk 1 above)
Comment (darshandamre) [reply style.css:64]: i don't have utility classes for flex properties, thank you for your time.

**Pull request merged**